packages/server/src/api/app/bsky/updateProfile.ts

```diff
@@ -87,11 +87,13 @@ export default function (server: Server) {
       .where('uri', '=', uri.toString())
       .execute()
 
-    const delBadgesQuery = db.db
-      .deleteFrom('app_bsky_profile_badge')
-      .where('profileUri', '=', uri.toString())
-      .where('badgeUri', 'in', toDelete)
-      .execute()
+    const delBadgesQuery = toDelete.length
+      ? db.db
+          .deleteFrom('app_bsky_profile_badge')
+          .where('profileUri', '=', uri.toString())
+          .where('badgeUri', 'in', toDelete)
+          .execute()
+      : null
```

Comment:
Not sure how/why I didn't see this issue until now, but on postgres an empty `toDelete` list was causing a syntax error.